cmd/yurt-manager/app/options/options.go

```diff
@@ -24,18 +24,20 @@ import (
 
 // YurtManagerOptions is the main context object for the yurt-manager.
 type YurtManagerOptions struct {
-	Generic            *GenericOptions
-	NodePoolController *NodePoolControllerOptions
-	GatewayController  *GatewayControllerOptions
+	Generic                       *GenericOptions
+	NodePoolController            *NodePoolControllerOptions
+	GatewayController             *GatewayControllerOptions
+	PoolCoordinatorCertController *PoolCoordinatorCertControllerOptions
 }
 
 // NewYurtManagerOptions creates a new YurtManagerOptions with a default config.
 func NewYurtManagerOptions() (*YurtManagerOptions, error) {
 
 	s := YurtManagerOptions{
-		Generic:            NewGenericOptions(),
```

Comment:
Default of field `LeaderElectionNamespace` in `Generic` is `kube-system`, so we also need to adapt this filed for `WorkingNamespace` parameter.